lib/route.js

```diff
@@ -363,9 +358,7 @@ exports = module.exports = internals.Route = class {
             return;
         }
 
-        if (this.method[0] !== '_') {
```

Comment:
This conditional is no longer needed, and having it here reduced the code-coverage.

Reply:
Is this actually not needed anymore, or just not exercised by the tests anymore?

Reply:
I haven't been able to find a code-path that relies on this branch, but I have to admit I'm not fluent in Hapi's codebase.

Reply:
The method used for the internal not found, bad request, and cors preflight routes is `_special`.  I think we'll want to keep this around and find a way to exercise it with one of those routes.

Reply:
You all were right. I found a way to exercise this code-path with https://github.com/hapijs/hapi/pull/4123/commits/3dc68cc90f713e35e72dcb23a84665c3a2778ca7